stdlib/2and3/zipfile.pyi

```diff
@@ -141,6 +141,25 @@ class ZipInfo:
         def is_dir(self) -> bool: ...
     def FileHeader(self, zip64: Optional[bool] = ...) -> bytes: ...
 
+if sys.version_info >= (3, 8):
+    class Path:
+        name: str
+        def __init__(self, root: Union[ZipFile, _Path, IO[bytes]], at: _SZI = ...) -> None: ...
```

Comment:
Are you sure `at: _SZI`? The code has code like `self.at.rstrip("/")` which suggest it can only be a string.

Reply:
Thanks for catching! I was going off of https://github.com/python/cpython/blob/3.8/Lib/zipfile.py#L2258